ocs_ci/ocs/resources/ocs.py

```diff
@@ -109,6 +110,15 @@ def delete(self, wait=True, force=False):
             bool: True if deleted, False otherwise
 
         """
+        # Avoid accidental delete of default storageclass and secret
+        if (
+            self.name == constants.DEFAULT_SECRET
+            or self.name == constants.DEFAULT_STORAGECLASS_CEPHFS
+            or self.name == constants.DEFAULT_STORAGECLASS_RBD
+        ):
+            log.info(f"Attempt to default default Secret or StorageClass")
```

Comment:
Is there a typo here ?

log.info(f"Attempt to default **to** default Secret or StorageClass")

Reply:
will fix it. thanks

Reply:
@vasukulkarni still needs to be fixed